contracts/Blacklistable.sol

```diff
@@ -41,7 +41,10 @@ contract Blacklistable is Ownable {
      * @dev Throws if called by any account other than the blacklister
      */
     modifier onlyBlacklister() {
-        require(msg.sender == blacklister);
+        require(
+            msg.sender == blacklister,
+            "Blacklistable: caller is not the blacklister"
```

Comment:
The format and wording of the error messages follow the convention in OpenZeppelin code.